### PR TITLE
Update manage-external-access.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5087
Added note and info.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-external-access.md [hunk 1]
```diff
@@ -41,7 +44,7 @@ Use external access when:
 
 ## Plan for external access
 
-By default, external access is turned on in Teams, which means that your organization can communicate with all external domains. If you add blocked domains, all other domains will be allowed; and if you add allowed domains, all other domains will be blocked. There are three scenarios for setting up external access in the Teams admin center (**Org-wide settings** > **External access**):
+By default, external access is turned on in Teams, which means that your organization can communicate with all external domains. If you add blocked domains, all other domains will be allowed; and if you add allowed domains, all other domains will be blocked. The exception to this rule is for meetings, if anonymous participants are allowed. There are three scenarios for setting up external access in the Teams admin center (**Org-wide settings** > **External access**):
```
Comment (JohanFreelancer9) [inline Teams/manage-external-access.md:47]:
```suggestion
By default, external access is turned on in Teams, which means that your organization can communicate with all external domains. If you add blocked domains, all other domains will be allowed; and if you add allowed domains, all other domains will be blocked. The exception to this rule is if anonymous participants are allowed in meetings. There are three scenarios for setting up external access in the Teams admin center (**Org-wide settings** > **External access**):
```

Comment (JohanFreelancer9) [reply Teams/manage-external-access.md:47]: Suggestion only.

**Comment by LolaJacobsen** [comment]
@vinbelms could you review this PR and let me know whether it's OK to merge?  Thanks!

**Pull request merged**